client/consensus/aura/Cargo.toml

```diff
@@ -37,6 +37,7 @@ sp-timestamp = { version = "2.0.0", path = "../../../primitives/timestamp" }
 sp-keystore = { version = "0.8.0", path = "../../../primitives/keystore" }
 sc-telemetry = { version = "2.0.0", path = "../../telemetry" }
 prometheus-endpoint = { package = "substrate-prometheus-endpoint", path = "../../../utils/prometheus", version = "0.8.0"}
+activate-wasm-bindgen-features = { version = "0.1.0", path = "../../../utils/activate-wasm-bindgen-features", optional = true }
```

Comment:
Please no. You add a dependency to a crate that is set to `publish = no`.

Than just remove aura completely from the wasm test compilation.

Or just build the node-template.

Reply:
> Or just build the node-template.

this requires some shenanigans to make it work, I'd like to leave it out of scope of this PR

went with a simple approach of adding one optional dependency to aura